portality/settings.py

```diff
@@ -584,14 +584,53 @@
     "issn" :  "index.issn.exact"
 }
 
+# =========================================
+# scheduled reports configuration
+REPORTS_BASE_DIR = "/home/cloo/reports/"
+REPORTS_EMAIL_TO = ["feedback@doaj.org"]
+
 # ========================================
 # Google Analytics configuration
 # specify in environment .cfg file - avoids sending live analytics
 # events from test and dev environments
 GOOGLE_ANALYTICS_ID = ''
 
+# Google Analytics custom dimensions. These are configured in the GA interface.
+GA_DIMENSIONS = {
+    'oai_res_id': 'dimension1',                                                                    # In GA as OAI:Record
+}
 
-# =========================================
-# scheduled reports configuration
-REPORTS_BASE_DIR = "/home/cloo/reports/"
-REPORTS_EMAIL_TO = ["feedback@doaj.org"]
+# GA for OAI-PMH
+OAI_GA_CATEGORY = 'OAI-PMH'
```

Comment:
I'd have personally named these GA_CATEGORY_* rather than *_GA_CATEGORY. I'm totally ambivalent about this though, so feel free to leave as is, just providing a different point of view.

Reply:
Evidently not completely ambivalent :)

I've often gone back and forth on this - do I want the differentiated text to be first (better for autofill) or last, which I guess is better for categorisation and understanding.

if you have ```dog_collie``` and ```dog_boxer``` is that more understandable than ```collie_dog``` or ```boxer_dog```? Easier to search for, since you can pick your choice of dog when you've half typed it? hmmmmmm

Reply:
Ambivalent in the sense that I mentioned it to have another point of view but you've obviously thought about it :)